### PR TITLE
Update Blueprint documentation to the last JHipster practices with blueprints

### PR DESCRIPTION
**Pull request by avdev4j**

### CODE REVIEW

File: modules/creating_a_blueprint.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ A JHipster blueprint:
 - is an NPM package, and is a Yeoman generator.
 - follows an extension of the Yeoman rules listed at [http://yeoman.io/generators/](http://yeoman.io/generators/) and can be installed, used and updated using the `yo` command. Instead of being prefixed by `generator-`, it is prefixed by `generator-jhipster-`, and instead of having just the `yeoman-generator` keyword, it must have 2 keywords, `yeoman-generator` and `jhipster-blueprint`.
 - A blueprint can only extend the following sub-generators (under the generators folder)
+    - app
```
Comment (ruddell) [inline modules/creating_a_blueprint.md:58]: We can also extend `heroku`, `cypress`, `page`, `entities`, and `ci-cd` subgenerators 

`ci-cd` is only in `main` branch and not in the beta releases yet.

https://github.com/jhipster/generator-jhipster/search?p=1&q=extends+BaseBlueprintGenerator

Comment (avdev4j) [reply modules/creating_a_blueprint.md:58]: awesome, was not aware of that. I will add it to the documentation but also in the blueprint generator.

Comment (avdev4j) [reply modules/creating_a_blueprint.md:58]: done in both doc PR but I will add a bullet point in the main ticket

**Comment by murdos** [comment]
@avdev4j : there's a `jhipster_7` branch, you should probably target it for this PR ;)

**Comment by avdev4j** [comment]
@murdos this PR will be compatible with the current version of JHipster, but for the other PR, Yes I will change it. thanks

File: modules/creating_a_blueprint.md [hunk 2]
```diff
@@ -65,6 +66,10 @@ A JHipster blueprint:
     - languages
     - spring-controller
     - spring-service
+    - heroku
+    - cypress
+    - page
+    - entities
```
Comment (ruddell) [inline modules/creating_a_blueprint.md:72]: I'm sorry I should have been more clear in my last comment.

`entities`, `cypress`, and `page` are only in the beta v7 versions and not v6.  `heroku` is in v6

Comment (avdev4j) [reply modules/creating_a_blueprint.md]: done ;)

Comment (avdev4j) [reply modules/creating_a_blueprint.md]: added app and heroku for the v6 and the other for the v7

Comment (avdev4j) [reply modules/creating_a_blueprint.md]: I think it's correct now, if it is let's merge this pr an will update the v7 and we will be ok with the doc

**Pull request merged**